clang/runtime/dpct-rt/include/dpct/graph.hpp

```diff
@@ -22,6 +22,8 @@ typedef sycl::ext::oneapi::experimental::command_graph<
     sycl::ext::oneapi::experimental::graph_state::executable>
     *command_graph_exec_ptr;
 
+typedef std::shared_ptr<sycl::ext::oneapi::experimental::node> node_ptr;
```

Comment:
As raw pointer is used for graph, I suggest using raw pointer here too.

Reply:
Done.